github_pr.py

```diff
@@ -104,7 +114,7 @@ def github_list_prs(**args):
     else:
         prs = repo.get_pulls()
         _print_prs(prs, **args)
-
+    return what_we_return
```

Comment:
this looks weirdly green highlighted to me - did the github webUI editor insert a tab here instead of spaces?